cpp/example_code/transcribe/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# CMakeLists.txt
+cmake_minimum_required(VERSION 3.11)
+set(CMAKE_CXX_STANDARD 11)
+project(demo LANGUAGES CXX)
+
+
+
+# Build shared libraries by default.
+if(NOT BUILD_SHARED_LIBS)
+    set(BUILD_SHARED_LIBS ON)
+endif()
+
+list(APPEND CMAKE_PREFIX_PATH "C:\\Program Files (x86)\\aws-cpp-sdk-all\\lib\\cmake")
+#Set the location of where Windows can find the installed libraries of the SDK.
+if(MSVC)
+    string(REPLACE ";" "/aws-cpp-sdk-all;" SYSTEM_MODULE_PATH "${CMAKE_SYSTEM_PREFIX_PATH}/aws-cpp-sdk-all")
+    list(APPEND CMAKE_PREFIX_PATH ${SYSTEM_MODULE_PATH})
+endif()
+
+
+find_package(AWSSDK COMPONENTS transcribe;transcribestreaming)
+# If the compiler is some version of Microsoft Visual C++,
+# and building as shared libraries, then dynamically link to those shared libraries.
+if(MSVC AND BUILD_SHARED_LIBS)
+    add_definitions(-DUSE_IMPORT_EXPORT)
+    # Copy relevant AWS SDK for C++ libraries into the current binary directory for running and debugging.
+    list(APPEND SERVICE_LIST transcribe;transcribestreaming)
+    
+    #For IDEs like Xcode and Visual Studio this line will be ignored because Release/Debug
+    #  is switched internally, but this is necessary for non-IDE builds.
+    set(CMAKE_BUILD_TYPE Debug) #TODO: Set to your build type
+	
+    #TODO:Choose appropriate one of the following two lines, you want to copy to the same folder where your executables are.
```

Comment:
#TODO:Select and use one of the following lines. Select the line that copies to the same folder where your executables are.